docs/meshstack.index.md

```diff
@@ -2,13 +2,22 @@
 id: meshstack.index
 title: Overview
 ---
+<h3 style="margin-top: 0">Welcome to the meshStack Documentation!</h3>
 
-Welcome to the Meshstack Documentation!
+meshStack is the name of our technology that powers the meshcloud solution. This document is targeted at cloud architects and SREs and introduces the components of meshStack and their interaction.
 
-Meshstack is the name of our technology that powers the Meshcloud solution. This document is targeted at cloud architects and SREs and introduces the components of Meshstack and their interaction.
+It is a modular software platform solution that connects organizational processes like project and access management with cloud technology. It provides a unified administration interface to the organization while replicating configuration into attached cloud platforms.
 
-Meshstack is a modular software platform solution that connects organizational processes like project and access management with cloud technology. It provides a unified administration interface to the organization while replicating configuration into attached cloud platforms.
+It is NOT a meta-layer across clouds. Users access native cloud APIs for their deployments, but meshStack helps them to ensure a proper and common configuration across cloud technologies with additional functionality such as metering, billing, surveillance and others.
 
-It is NOT a meta-layer across clouds. Users access native cloud APIs for their deployments, but Meshstack helps them to ensure a proper and common configuration across cloud technologies with additional functionality such as metering, billing, surveillance and others.
+![meshStack Context and Components](assets/meshstack-chart.svg)
 
-![Meshstack Context and Components](assets/meshstack-chart.svg)
+## Technological Terms
```

Comment:
"Mapping of mesh business entities to cloud platform concepts" would be a more transparent heading

Reply:
Yes but its a bit long for a catchy header and probably looks ugly in the TOC at the side. I check and maybe I can shorten it a bit

Reply:
Called it `Mesh-Entity Mapping to Cloud Platform Concepts` as your suggestion is too long.